src/typings/global.ts

```diff
@@ -467,6 +476,13 @@ declare global {
     skuSpecifications?: SkuSpecification[]
   }
 
+  interface SearchItemAttribute {
+    id: string,
+    name: string,
+    value: string,
+    visible: boolean
+  }
+
```

Comment:
since the two types have the same properties, you can create just one so you don't need a function to convert the types